### PR TITLE
Add comments to functions

### PR DESCRIPTION
**Pull request by giusebar**

### CODE REVIEW

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 1]
```diff
@@ -98,12 +98,19 @@ def __init__(self, name):
         threading.Thread(None, rospy.spin)
 
     def set_planner_id(self, planner_id):
+        """
+        Sets the planner_id used for all future planning requests
+        @param planner_id - The string for the planner id, set to None to clearp
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: typo `clearp`

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 2]
```diff
@@ -98,12 +98,19 @@ def __init__(self, name):
...
+        """
         self._move_group_commander.set_planner_id(planner_id)
 
     def set_num_planning_attempts(self, num_planning_attempts):
         self._move_group_commander.set_num_planning_attempts(num_planning_attempts)
 
     def set_planning_time(self, seconds):
+        """
+        Set default planning time to be used for future planning request.
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Is it a maximum planning time? Does it have the same meaning regardless of the planner used?

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 3]
```diff
@@ -153,6 +182,9 @@ def execute(self):
             rospy.logwarn("No plans were made, not executing anything.")
 
     def execute_plan(self, plan):
+        """
+        Executes a given plan.
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py:200]: Please document the parameter `plan`

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 4]
```diff
@@ -118,9 +125,14 @@ def get_end_effector_pose_from_state(self, state):
         return response.pose_stamped[0]
 
     def get_planning_frame(self):
+        """ Get the name of the frame where all planning is performed """
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Please document the return value.
```python
"""
@return: this is a description of what is returned
"""
```

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 5]
```diff
@@ -789,6 +823,17 @@ def get_ik(self, target_pose, avoid_collisions=False):
             rospy.logerr("Service call failed: %s" % e)
 
     def move_to_pose_value_target_unsafe(self, target_pose,  avoid_collisions=False, time=0.002, wait=True):
+        """
+        Specify a target pose for the end-effector and moves to it
+        @param target_pose - new pose of end-effector: a Pose message, a PoseStamped
+        message or a list of 6 floats: [x, y, z, rot_x, rot_y, rot_z] or a list
+        of 7 floats [x, y, z, qx, qy, qz, qw]
+        @param avoid_collisions - Find an IK solution that avoids collisions. By default, this is false
+        @param - time in s (counting from now) for the robot to reach the
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Add name of the parameter

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 6]
```diff
@@ -131,15 +143,32 @@ def refresh_named_targets(self):
...
     def allow_looking(self, value):
+        """
+        Enable/disable looking around for motion planning
+        @param value - boolean
+        """
         self._move_group_commander.allow_looking(value)
 
     def allow_replanning(self, value):
+        """
+        Enable/disable replanning in case new obstacle are detected
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: typo `new obstacles`

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 7]
```diff
@@ -213,6 +245,7 @@ def named_target_in_srdf(self, name):
         return name in self._srdf_names
 
     def set_named_target(self, name):
+        """ Set a joint configuration by name. """
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Please use the same multiline format as elsewhere.
Document parameter `name`

Comment (toliver) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Document return value.

File: sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py [hunk 8]
```diff
@@ -233,6 +266,7 @@ def set_named_target(self, name):
         return True
 
     def get_named_target_joint_values(self, name):
+        """ Get the joint angles for targets specified by name. """
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Multiline

Comment (toliver) [reply sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py]: Document @return value

**Review by toliver — DISMISSED** [review]

**Review by kirstyellis — APPROVED** [review]

**Pull request merged**